app/models/recipient.rb

```diff
@@ -1,3 +1,7 @@
 class Recipient < ApplicationRecord
   has_many :complaints
+
+  def self.styles
+    Recipient.all
```

Comment:
I think we don't need to make a class method in this model to use another class method for this model, I suppose you want to do something else and this definition should not be here at all